client/my-sites/plans-v2/plans-column/index.tsx

```diff
@@ -102,6 +102,25 @@ const PlansColumn = ( { duration, onPlanClick, productType, siteId }: PlanColumn
 				...product,
 				owned: product.productSlug === currentPlan,
 				legacy: false,
+			} ) )
+			// If product offers 'realtime' and 'daily' options, AND product is not already owned,
+			// append "Available Options: Real-Time and Daily" to the product description.
+			.map( ( product ) => ( {
+				...product,
+				description:
+					product.subtypes?.filter( ( subtype ) => /_(daily|realtime)$/.test( subtype ) ).length >=
+						2 && product.productSlug !== currentPlan ? (
+						<>
+							{ product.description }{ ' ' }
+							{ translate( '{{em}}Available options: Real-Time or Daily.{{/em}}', {
```

Comment:
In the final copydeck, it's `Real-time`. This part could perhaps be extracted in a util function since it's used in both the product and plan column.

Reply:
I originally meant just extracting what's inside the `<>`, since that's what's duplicated, but that's even better! Have two different functions is perfectly fine to me.

I feel that `utils` is a more adequate place than `translations`. I've stumbled upon the same problem lately: in that case, instead of renaming `utlils.ts` to `utils.tsx` (which could cause some annoying conflicts for now indeed), you could use the React functions directly instead of the JSX syntax, e.g. `React.createElement(React.Fragment, {}, children)`.

Reply:
> you could use the React functions directly instead of the JSX syntax, e.g. React.createElement(React.Fragment, {}, children).

Oh yes, thats a great idea!  Will do, thanks!